manifests/init.pp

```diff
@@ -750,4 +761,14 @@
     validate_hash($keys_real)
     create_resources('ssh_authorized_key', $keys_real)
   }
+  # Validate SSHD Config Puppet 3.5 and newer uses validate_cmd of file type, puppet version before 3.5 use stdlib validate_cmd
+  # which uses the sshd binary on the master!
+  if $validate_sshd_config_real == true {
+    if versioncmp($::puppetversion, '3.5') >= 0 {
+      File['sshd_config'] {
+        validate_cmd => "${sshd_bin_path} -t -f %" }
```

Comment:
this should have a comma at the end and the closing curly brace should be on the next line